File: app/src/main/java/cosmic/com/pkwprj/model/GithubOwner.java
package cosmic.com.pkwprj.model

import com.google.gson.annotations.SerializedName

data class GithubOwner(@SerializedName("login")val login:String?=null,

                       @SerializedName("avatar_url")val avatar_url:String?=null,

                       @SerializedName("html_url")val html_url:String?=null,

                       @SerializedName("score")val score:Int?=null)

    constructor() {}

    constructor(login: String, avatar_url: String, html_url: String, score: Int) {
        this.login = login
        this.avatar_url = avatar_url
        this.html_url = html_url
        this.score = score
    }

File: app/src/main/java/cosmic/com/pkwprj/model/GithubOwner.kt
package cosmic.com.pkwprj.model

class GithubOwner {

    lateinit var login: String
    lateinit var avatar_url: String
    lateinit var html_url: String
    var score:Float = 0.0f

    constructor() {}

    constructor(login: String, avatar_url: String, html_url: String, score: Float) {
        this.login = login
        this.avatar_url = avatar_url
        this.html_url = html_url
        this.score = score
    }
}
